src/OrchardCore.Modules/OrchardCore.Lists/Views/ProfileHeader.cshtml

```diff
@@ -0,0 +1,11 @@
+@model OrchardCore.Lists.ViewModels.ProfileHeaderViewModel
+@inject OrchardCore.ContentManagement.Display.IContentItemDisplayManager ContentItemDisplayManager
+@inject OrchardCore.DisplayManagement.ModelBinding.IUpdateModelAccessor UpdateModelAccessor
+@{
+    var profileShape = await ContentItemDisplayManager.BuildDisplayAsync(Model.ContainerContentItem, UpdateModelAccessor.ModelUpdater, "Profile");
+}
+<div class="card bg-light mb-3">
```

Comment:
Does `bg-light` work on dark mode?

Reply:
@agriffard i change that so it works on the dark mode as well.